tests/unit/test_util.py

```diff
@@ -0,0 +1,23 @@
+from io import BytesIO, StringIO
+
+import pytest
+
+from vcr import request
+from vcr.util import read_body
+
+
+@pytest.mark.parametrize(
+    "input, output",
+    [
+        (BytesIO(b"Stream"), b"Stream"),
+        (StringIO("Stream"), b"Stream"),
+        (iter(["StringIter"]), b"StringIter"),
+        (iter([b"BytesIter"]), b"BytesIter"),
```

Comment:
Maybe the last two should (a) have more than one list element or (b) be duplicated so that we cover single and non-single cases. What do you think?

Reply:
done.